src/routes/index.ts

```diff
@@ -0,0 +1,69 @@
+import { Hono } from 'hono';
```

Comment:
1. Why user route is in `index.ts`?
2. Please create a separate file for a user route
3. `Index.ts` should accepts all the routes and export then with one instance